src/main/resources/bootstrap.yaml

```diff
@@ -0,0 +1,23 @@
+spring:
+  cloud:
+    propertiesvolume:
+      prefixed: false
+      paths: /mnt/secrets/rpe-ft-api
+      aliases:
+        admin-username-cmc: ADMIN_USERNAME_CMC
+        admin-password-cmc: ADMIN_PASSWORD_CMC
+        editor-username-cmc: EDITOR_USERNAME_CMC
+        editor-password-cmc: EDITOR_PASSWORD_CMC
+        admin-username-divorce: ADMIN_USERNAME_DIVORCE
+        admin-password-divorce: ADMIN_PASSWORD_DIVORCE
+        editor-password-divorce: EDITOR_PASSWORD_DIVORCE
+        editor-username-divorce: EDITOR_USERNAME_DIVORCE
+        admin-password-probate: ADMIN_PASSWORD_PROBATE
+        admin-username-probate: ADMIN_USERNAME_PROBATE
+        editor-password-probate: EDITOR_USERNAME_PROBATE
+        editor-username-probate: EDITOR_PASSWORD_PROBATE
+        admin-password-test: ADMIN_PASSWORD_TEST
+        admin-username-test: ADMIN_USERNAME_TEST
+        editor-password-test: EDITOR_PASSWORD_TEST
+        editor-username-test: EDITOR_USERNAME_TEST
+        rpe-ft-feature-toggle-api-POSTGRES-PASS: FEATURES_DB_PASSWORD
```

Comment:
do you want this by default?
won't it break the local setup using the postgres chart?